### DISCUSSION TITLE
Better Installation Steps ~ First Install & Deployment Help

### DISCUSSION THREAD

**Discussion by thefourcraft** [familytree365/genealogy#2087] [Q&A] [answered]
## What do I Want? 😁
I want to install the [genealogy](https://github.com/cgdprojects/genealogy) on my server
The README.md doesn't have enough installation information, as far as I got it it's a backed application, but it also is saying that it's a website.
also, the client part confused me initially, who suppose to use it?
If anyone can explain the installation process for the [genealogy](https://github.com/cgdprojects/genealogy) GitHub project that would be amazing.

Can this be installed on a web host server? or it must be a dedicated server?

## Translations 🌐
I would like to contribute by adding a translation for the Hebrew language, as I got it simply fork the repo and pull when I am done?

## Website SQL bug 🐛
seems like the main website https://www.familytree365.com/# has an SQL bug, probably lost the conction to the database
```sql
SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) (SQL: insert into `people` (`name`, `email`, `updated_at`, `created_at`) values (asd asd, me@thefourcraft.com, 2022-09-23 00:09:41, 2022-09-23 00:09:41))
```
happens when you try to create an account...

**Comment by tareqdokhan** [comment]
hello are you have the solution, I have a problem when I open it on the public path, its shows lazy loading and continues like a loop.

**Reply by thefourcraft** [reply]
I don't this project doesn't get any attention

**Reply by thefourcraft** [reply]
btw, if it's possible can you explain step by step what did you do to install it?

**Comment by curtisdelicata** [comment, marked as answer]
This repository is the backend api for the whole website you need to also install the client side for public access.

Demo was fixed a while ago.

Thank you for your interest.

**Comment by curtisdelicata** [comment]
We will finalise translations support soon too. Thank you for the offer, we will contact you when ready.
